Makefile

```diff
@@ -191,7 +191,7 @@ manually_upload_certs: check-tf-version ## Manually upload to AWS the SSL certif
 pingdom: check-tf-version ## Use custom Terraform provider to set up Pingdom check
 	$(if ${ACTION},,$(error Must pass ACTION=<plan|apply|...>))
 	$(eval export PASSWORD_STORE_DIR?=~/.paas-pass)
-	$(eval export PINGDOM_CONTACT_IDS=11089310)
+	$(eval export PINGDOM_CONTACT_IDS=11089310,11189971)
```

Comment:
@saliceti is planning to reconfigure the PagerDuty services to the 4 proposed by @mtekel on the whiteboard this morning. Should we take this opportunity now to use separate contact IDs (and thus escalation policies) for CI and staging?